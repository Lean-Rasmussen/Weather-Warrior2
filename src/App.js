import React, { Component } from 'react';
//import styles
import './App.css';
//import components
import NavBar from './components/navigationBar';
import Battle from './components/battle';
import Home from './components/home';
import NewChar from './components/newChar';

class App extends Component {
  render() {
    return (
      <div className="App">
        <h1> Gif warrior 2.0 </h1>
        <h3> time to get giffing</h3>
        <NavBar />
      </div>
    );
  }
}

export default App;