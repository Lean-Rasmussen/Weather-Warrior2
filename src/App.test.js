import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { shallow } from 'enzyme';


const app = shallow(<App />);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});



it('Renders correctly', ()=> {
    expect(app).toMatchSnapshot();
})