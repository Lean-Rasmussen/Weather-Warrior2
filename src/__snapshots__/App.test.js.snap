// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
           Gif warrior 2.0 
</h1>,
        <h3>
           time to get giffing
</h3>,
        <NavBar />,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Gif warrior 2.0 ",
        },
        "ref": null,
        "rendered": " Gif warrior 2.0 ",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " time to get giffing",
        },
        "ref": null,
        "rendered": " time to get giffing",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
             Gif warrior 2.0 
</h1>,
          <h3>
             time to get giffing
</h3>,
          <NavBar />,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Gif warrior 2.0 ",
          },
          "ref": null,
          "rendered": " Gif warrior 2.0 ",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " time to get giffing",
          },
          "ref": null,
          "rendered": " time to get giffing",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
